module/zstd/lib/zstd.c

```diff
@@ -21273,7 +21273,7 @@ ZSTD_initStats_ultra(ZSTD_matchState_t* ms,
     ms->window.lowLimit = ms->window.dictLimit;
     ms->nextToUpdate = ms->window.dictLimit;
 
-    /* re-inforce weight of collected statistics */
+    /* re-enforce weight of collected statistics */
```

Comment:
this one isn't clear to me which way it should be... are we "enforcing" some sort of rule? or are we "reinforcing" (supplementing, increasing) the weight?  Maybe better to chose a different word altogether.

Reply:
We should prefer making any changes to zstd.c and zstd.h through upstream.